src/script/page/state.ts

```diff
@@ -52,7 +80,152 @@ type AppMainState = {
   };
 };
 
+const legalHoldModalDefaultState = {
+  conversation: null,
+  conversationId: '',
+  fingerprint: '',
+  isLoading: false,
+  isOpen: false,
+  isRequestModal: false,
+  isSelfInfo: false,
+  skipUsers: false,
+  type: null,
+  users: [],
+};
+
 const useAppMainState = create<AppMainState>((set, get) => ({
+  legalHoldModal: {
+    closeModal: () =>
+      set(state => ({
+        ...state,
+        legalHoldModal: {
+          ...state.legalHoldModal,
+          ...legalHoldModalDefaultState,
```

Comment:
im a little new to zustand but shouldn't the incoming state com after the defaultState?